src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -54,14 +52,11 @@ class SecurityExtension extends Extension implements PrependExtensionInterface
     private $requestMatchers = [];
     private $expressions = [];
     private $contextListeners = [];
-    /** @var array<array{0: int, 1: AuthenticatorFactoryInterface|SecurityFactoryInterface}> */
+    /** @var list<array{0: int, 1: AuthenticatorFactoryInterface}> */
```

Comment:
why list instead of array? what's the diff?

Reply:
> Psalm supports a `list` type that represents continuous, integer-indexed arrays like `["red", "yellow", "blue"]` .
>
> These arrays have the property `$arr === array_values($arr)`, and represent a large percentage of all array usage in PHP applications.

(where "Psalm supports" is generic support across PHP static analysers)